week2/homework/src/todoList.js

```diff
@@ -0,0 +1,66 @@
+const fs = require('fs');
+
+const config = require('./config')
+
+const newTodos = new WeakMap();
+const newRead = new WeakMap();
+const newWrite = new WeakMap();
```

Comment:
My WeakMap knowledge is limited. Can you explain me why do you need three different maps?